WalletWasabi.Fluent/ViewModels/Wallets/Send/SendViewModel.cs

```diff
@@ -71,6 +73,11 @@ public SendViewModel(UiContext uiContext, WalletViewModel walletVm)
 
 		Balance = walletVm.WalletModel.Balances;
 
+		_suggestionLabels = new SuggestionLabelsViewModel(WalletVm.WalletModel, Intent.Send, 3)
+		{
+			Labels = { }
+		};
```

Comment:
Initializing `Labels` is not required here:

```suggestion
		_suggestionLabels = new SuggestionLabelsViewModel(WalletVm.WalletModel, Intent.Send, 3);
```